llms/mlx_lm/server.py

```diff
@@ -433,8 +446,10 @@ def handle_chat_completions(self) -> mx.array:
             hasattr(self.tokenizer, "apply_chat_template")
             and self.tokenizer.chat_template
         ):
+            messages = body["messages"]
+            messages = openai_to_common_denominator(messages)
```

Comment:
this could be behind a flag?

Reply:
Yes, maybe it is better to put it behind a flag for now. I left a comment elsewhere.. I'm still a little confused by it tbh. But at least if it's behind a flag it won't break anything by default.